### PR TITLE
travis: use from-source fanuc deps.

### PR DESCRIPTION
**Pull request by gavanderhoorn**
As ros-industrial/fanuc has not yet been released for Melodic.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -22,8 +22,8 @@ env:
   matrix:
     - ROS_DISTRO="kinetic"
     - ROS_DISTRO="kinetic" ROS_REPO=ros-shadow-fixed
-    - ROS_DISTRO="melodic"
-    - ROS_DISTRO="melodic" ROS_REPO=ros-shadow-fixed
+    - ROS_DISTRO="melodic" UPSTREAM_WORKSPACE='github:ros-industrial/fanuc#indigo-devel'
+    - ROS_DISTRO="melodic" UPSTREAM_WORKSPACE='github:ros-industrial/fanuc#indigo-devel' ROS_REPO=ros-shadow-fixed
```
Comment (gavanderhoorn) [inline .travis.yml:20]: @ipa-mdl: is `UPSTREAM_WORKSPACE` like this only supported in `master` of `industrial_ci`?

Comment (mathias-luedtke) [reply .travis.yml:20]: Yes.
The legacy version only supports .rosinstall URLs/files.

Comment (gavanderhoorn) [reply .travis.yml:20]: Hm. 1-0 for `master` then.

I'll see if I'll switch.

**Comment by gavanderhoorn** [comment]
Closing as I've merged this in as d09f5ff209e24de51339b7c8fedf03442d8aeb37.

**Pull request closed without merging**